### ISSUE TITLE
Current cmsearch missing required options

### ISSUE THREAD

**Issue by chofski** [ppgardne/RNIE#1]
Using the latest Infernal package leads to an error from RNIE due to the "--fil-no-qdb" option not being supported. Would suggest updating docs to highlight specific version of Infernal to use or update Perl script to handle these changes.

**Comment by ppgardne** [comment]
Thanks for the reminder Thomas. 
I've been meaning to update the docs for a while. Unfortunately I haven't got around to reparameterising for the latest versions of Infernal.

**Comment by NickNCL** [comment]
I ran into the same issue and was able to run RNIE with infernal version 1.0 installed

**Comment by ppgardne** [comment]
Yep, Infernal 1.0 is what was used for optimising the terminator annotations. It would take some work to perform a similar optimisation with the latest Infernal release.
